modules/platform/nuxeo-platform-document-routing/nuxeo-routing-api/src/main/java/org/nuxeo/ecm/platform/routing/api/DocumentRoutingService.java

```diff
@@ -622,4 +622,19 @@ void grantPermissionToTaskDelegatedActors(CoreSession session, String permission
      */
     boolean isWorkflowModel(final DocumentRoute documentRoute);
 
+    /**
+     * Returns true if the workflowModel can be started on the document list
+     * @param documentIds
+     * @param worflowModelName
+     * @since 11.3
+     */
+    boolean canCreateInstance(CoreSession session, List<String> documentIds, String worflowModelName);
+
+    /**
+     * Returns a list of runnable DocumentRoute for the input document IDs
```

Comment:
`Returns a list of runnable document routes for the input document IDs.`

Reply:
done